package-lock.json

```diff
@@ -12222,18 +12225,21 @@
         },
         "anafanafo": {
           "version": "1.0.0",
-          "bundled": true,
+          "resolved": "https://registry.npmjs.org/anafanafo/-/anafanafo-1.0.0.tgz",
+          "integrity": "sha512-pDPbI7SFRHu0byMXHAf+v74+LCcHSxnLYkcbfiV91XRlE+NSLpFCpEQdVUy9ZxZw/LuhTrOin4r8wlR3OFrKBA==",
```

Comment:
Interesting, when I ran `npm install` it did change something related to anafanafo. I'm on npm 6.9.0, how about you?

Reply:
6.2.0

Reply:
Hmm, you might want to upgrade. It looks like this is deploying fine.